values/otomi-api/otomi-api.gotmpl

```diff
@@ -46,7 +46,7 @@ env:
   CUSTOM_ROOT_CA: |
     {{- $v._derived.caCert | nindent 4 }}
   {{- end }}
-  IN_DOCKER: '1'
+  IN_DOCKER: 'false'
```

Comment:
I believe that IN_DOCEKR should be set here.

Reply:
I replaced all the `IN_DOCKER=1` with `IN_DOCKER=false`... So setting this to `true` would actually change the logica indicating that the previous `IN_DOCKER: '1'` was wrong.